library/cartridge_apply_app_config.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+
+import json
+import os
+import zipfile
+
+from ansible.module_utils.helpers import Helpers as helpers
+from ansible.module_utils.urls import open_url, basic_auth_header
+
+argument_spec = {
+    'local_config_path': {'required': True, 'type': 'str'},
+    'remote_dir': {'required': False, 'type': 'str', 'default': '/tmp/'},
+    'upload_endpoint': {'required': False, 'type': 'str'},
+    'console_sock': {'required': False, 'type': 'str'},
+    'http_port': {'required': False, 'type': 'int'},
+    'cluster_cookie': {'required': False, 'type': 'str'},
+    'tdg_token': {'required': False, 'type': 'str'},
+}
+
+LUA_ENDPOINT = 'lua'
+HTTP_ENDPOINT = 'http'
+TDG_ENDPOINT = 'tdg'
```

Comment:
Why do you name it `endpoint`? It looks like it HTTP endpoint for config uploading. What about `mode`?

Reply:
It was the first version of naming, in doc I used `mode` word)